packages/node_modules/@ciscospark/plugin-conversation/src/conversation.js

```diff
@@ -469,10 +469,16 @@ const Conversation = SparkPlugin.extend({
   /**
    * Creates a ShareActivty for the specified conversation
    * @param {Object} conversation
+   * @param {Object} activity
    * @returns {ShareActivty}
    */
-  makeShare(conversation) {
-    return ShareActivity.create(conversation, null, this.spark);
+  makeShare(conversation, activity) {
```

Comment:
Why can't the result of `makeShare()` *be* the provisional activity? When josh and I original talked about `makeShare()`, I'm pretty sure that was the plan and that was why we intentionally left the activity param out of `makeShare()`